pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -579,7 +579,16 @@ cdef class _Timestamp(ABCTimestamp):
     @property
     def is_month_start(self) -> bool:
         """
-        Return True if date is first day of month.
+        Return True if the date is the first day of the month.
+
+        Returns
+        -------
+        bool
+            True if the date is the first day of the month.
```

Comment:
This is kinda redundant. Thoughts about writing Check if date is first day of month as the first sentence? Not sure...

Reply:
I agree, I'll fix it, thank you